delta/app/src/main/resources/app.conf

```diff
@@ -116,23 +116,6 @@ app {
     resolve-base = "http://localhost:8081"
   }
 
-  # Json-LD Api configuration
-  json-ld-api {
```

Comment:
The ability to have to configure the severity of the parser was introduced a while back to avoid problems down the line with Blazegraph for example because some `@id` would be invalid.

Since then it was always used as strict and we never had problems so we can just get rid of it